src/Data/Api.elm

```diff
@@ -0,0 +1,47 @@
+module Data.Api exposing
+    ( getListOfAudienceFolders
+    , getListOfAudiences
+    )
+
+import Data.Audience as Audience exposing (Audience)
+import Data.AudienceFolder as AudienceFolder exposing (AudienceFolder)
+import Http
+import Json.Decode as Decode exposing (decodeString)
+import Process
+import Task exposing (Task)
```

Comment:
I like those imports (exposing types and single word qualification). this is what we do as well.